clusterctl/cmd/create_cluster.go

```diff
@@ -60,14 +63,20 @@ func RunCreate(co *CreateOptions) error {
 		return err
 	}
 
-	d := &clusterdeployer.ClusterDeployer{}
+	mini := minikube.New(co.VmDriver)
+	d := clusterdeployer.New(mini, co.CleanupExternalCluster)
 	err = d.Create(c, m)
 	return err
 }
 
 func init() {
+	// Required flags
 	createClusterCmd.Flags().StringVarP(&co.Cluster, "cluster", "c", "", "A yaml file containing cluster object definition")
 	createClusterCmd.Flags().StringVarP(&co.Machine, "machines", "m", "", "A yaml file containing machine object definition(s)")
+
+	// Optional flags
+	createClusterCmd.Flags().BoolVarP(&co.CleanupExternalCluster, "cleanup-external-cluster", "", true, "Whether to cleanup the external cluster after bootstrap")
```

Comment:
Do you imagine that this would be set to false mostly for debugging purposes, or do you think there are cases where people would want to keep the external cluster around long term?

Reply:
For minikube, I would say it is mostly for debugging. However, if a different and more persistent external cluster is used (eg. a GKE, EKS) I could imagine not deleting the external cluster and not pivoting cluster objects.